docs_src/microservices/support/cronScheduler/details/DataRetention.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Support Cron Scheduler - Data Retention and Persistent Caps
+---
+
+# Support Cron Scheduler - Data Retention and Persistent Caps
+
+## Overview
+
+### Support Cron Scheduler service
+In use cases, since support-cron-scheduler service persists data in the local database indefinitely, there is a need to persist the latest recent schedule action records (hereafter referred to as records) only and clean up the old ones, as keeping an infinite number of records is considered computationally expensive and can lead to a lack of machine memory. Thus, a retention mechanism is placed on support-cron-scheduler to keep a certain number of records.
```

Comment:
```suggestion
In use cases, since support-cron-scheduler service persists data in the local database indefinitely, there is a need to persist the latest recent schedule action records (hereafter referred to as records) only and clean up the old ones, as keeping an infinite number of records is considered computationally expensive and can lead to a lack of machine storage. Thus, a retention mechanism is placed on support-cron-scheduler to keep a certain number of records.
```